### DISCUSSION TITLE
RFE: is it possible to start making github releases?🤔

### DISCUSSION THREAD

**Discussion by kloczek** [Yubico/libfido2#745] [Ideas]
On create github release entry is created email notification to those whom have set in your repo the web UI Watch->Releases.
gh release can contain additional comments (li changelog) or additional assets like release tar balls (by default it contains only assets from git tag) however all those part are not obligatory.
In simplest variant gh release can be empty because subiekt of the sent email contains git tag name.

I'm asking because my automation process uses those email notifications by trying to make preliminary automated upgrades of building packages, which allows saving some time on maintaining packaging procedures.
Probably other people may be interested to be instantly informed about release new version as well.

Documentation and examples of generate gh releases:
https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
https://cli.github.com/manual/gh_release_upload/
https://github.com/jbms/sphinx-immaterial/pull/282
https://github.com/marketplace/actions/github-release
https://pgjones.dev/blog/trusted-plublishing-2023/
https://github.com/jbms/sphinx-immaterial/issues/281#issuecomment-1700933026
tox target to publish on pypi and make gh release https://github.com/jaraco/skeleton/blob/928e9a86d61d3a660948bcba7689f90216cc8243/tox.ini#L42-L58

**Comment by LDVG** [comment]
Hi,

This project currently has no plan to start using GitHub Releases. We publish our signed tarballs to [Yubico's release page](https://developers.yubico.com/libfido2/Releases/). You can probably create some automation to watch this page for updates; [Debian does this through uscan](https://salsa.debian.org/auth-team/libfido2/-/blob/6ea4d7dd25f0ee3638dc8b42cd22a622dc5532d3/debian/watch), for example. 

We will keep your request in mind when reviewing our release processes in the future. :slightly_smiling_face:

**Comment by kloczek** [comment]
Again: this RFE has nothing to do with any files (signed or not).

**Reply by LDVG** [reply]
Thank you for the clarification. I'll reiterate my response above: we'll keep this in mind for the future.

**Comment by kloczek** [comment]
Thank you very much 😄 👍
